torch/_dynamo/variables/functions.py

```diff
@@ -30,16 +33,38 @@ def wrap_bound_arg(val, options):
         return variables.EnumVariable(val, **options)
     elif isinstance(val, (type, abc.ABCMeta)):
         return variables.UserDefinedClassVariable(val, **options)
+    elif istensor(val):
```

Comment:
I hate this util

Reply:
why, is it dangerous? should i be specific about which tensor classes i support here?